lib/matplotlib/widgets.py

```diff
@@ -627,6 +627,9 @@ class RadioButtons(AxesWidget):
 
      *circles*
         A list of :class:`matplotlib.patches.Circle` instances
+        
+     *val*
```

Comment:
FWIW - I don't see a good reason not to be very explicit about the name of the variable. ``value`` would be good, ``selected_value`` would be even better...